### PR TITLE
PMM-5732 2.5.0 Release Notes

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: source/release-notes/2.5.0.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* :jirabug:`PMM-4252`: The resolution of the PMM Server favicon image has been improved.
+* :jirabug:`PMM-5581`: PMM Server Grafana plugins can now be updated on the command line with the ``grafana-cli`` command-line utility.
+* :jirabug:`PMM-5536`: Three Grafana plugins were updated to the latest versions: vertamedia-clickhouse-datasource to 1.9.5, grafana-polystat-panel to 1.1.0, and grafana-piechart-panel to 1.4.0.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PMM-5351`: The documentation on MongoDB user privileges has been corrected SKIP
```
Comment (shoffman-percona) [inline source/release-notes/2.5.0.rst:35]: Heh...looks like someone put "SKIP" in the "extra comments field of Jira.  Needs to be removed.

Comment (PaulJacobs-percona) [reply source/release-notes/2.5.0.rst]: Done. How does one 'SKIP'?

Comment (shoffman-percona) [reply source/release-notes/2.5.0.rst]: I assumed whoever did it put "SKIP" in the text field thinking there was nothing needed and perhaps we have some validation that would make that a required field in some situations?

Comment (PaulJacobs-percona) [reply source/release-notes/2.5.0.rst]: We were.

**Pull request merged**